### PR TITLE
Update "Resolved" styling for code review comments

### PR DESCRIPTION
**Pull request by jmkulwik**
This updates the styling of code review comments to be the "expanded" version of the expand/collapse comment.

Up next: Support collapsing and expanding comments

The new styling

Note: The teacher name is still displayed in blue in order to visually differentiate from a student.

## Links

- jira ticket: [CSA-640](https://codedotorg.atlassian.net/browse/CSA-640)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/codeReview/CommentOptions.jsx [hunk 1]
```diff
@@ -19,14 +19,14 @@ class CommentOptions extends Component {
   commentOptionTypes = {
     resolve: {
       onClick: this.props.onResolveStateToggle,
-      iconClass: 'fa fa-fw fa-check',
-      text: javalabMsg.resolve(),
```
Comment (bencodeorg) [inline apps/src/templates/instructions/codeReview/CommentOptions.jsx:23]: Do we never remove strings from translation?

**Review by bencodeorg — APPROVED** [review]

File: apps/src/templates/instructions/codeReview/CommentOptions.jsx (hunk 1 above)
Comment (jmkulwik) [reply apps/src/templates/instructions/codeReview/CommentOptions.jsx:23]: Hmm I hadn't heard that before, but that doesn't mean it's not true. Do you have more context?

Comment (bencodeorg) [reply apps/src/templates/instructions/codeReview/CommentOptions.jsx:23]: I for some reason misread that you didn't delete the old strings (you did). Ignore me!

File: apps/src/templates/instructions/codeReview/Comment.jsx [hunk 2]
```diff
@@ -89,17 +89,18 @@ class Comment extends Component {
       <div
         style={{
           ...styles.commentContainer,
-          ...(isFromOlderVersionOfProject &&
-            styles.olderVersionCommentTextColor)
+          ...((isFromOlderVersionOfProject || isResolved) && styles.lessVisible)
```
Comment (sanchitmalhotra126) [inline apps/src/templates/instructions/codeReview/Comment.jsx:92]: nit: might be worth moving `isFromOlderVersionOfProject || isResolved` to a local variable

**Review by sanchitmalhotra126 — APPROVED** [review]
LGTM!

File: apps/src/templates/instructions/codeReview/Comment.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/templates/instructions/codeReview/Comment.jsx:92]: I should note that `isFromOlderVersionOfProject` is always false b/c we never implemented this:

https://github.com/code-dot-org/code-dot-org/blob/76bc1de4f96e2003ba9b150ec9b659f567de5f67/dashboard/app/controllers/code_review_comments_controller.rb#L97

I'll add a tech debt item for this to delete references to it (or is this something we're planning on implementing @jmkulwik)?

Comment (jmkulwik) [reply apps/src/templates/instructions/codeReview/Comment.jsx:92]: It is something we plan to implement eventually (after launch), but probably not something we should be maintaining currently.

Comment (bencodeorg) [reply apps/src/templates/instructions/codeReview/Comment.jsx:92]: Got it, added to tech debt epic here:

https://codedotorg.atlassian.net/browse/CSA-1020

File: apps/src/templates/instructions/codeReview/CommentOptions.jsx [hunk 3]
```diff
@@ -19,14 +19,14 @@ class CommentOptions extends Component {
...
+      iconClass: 'fa fa-fw fa-check-circle',
+      text: javalabMsg.markComplete(),
       key: 'Resolve'
```
Comment (molly-moen) [inline apps/src/templates/instructions/codeReview/CommentOptions.jsx:24]: nit: should we change this key (and the re-open key below)?

**Review by molly-moen — APPROVED** [review]
LGTM, just one nit

File: apps/src/templates/instructions/codeReview/CommentOptions.jsx (hunk 3 above)
Comment (jmkulwik) [reply apps/src/templates/instructions/codeReview/CommentOptions.jsx:24]: I'll go ahead and update this in the next PR.

**Pull request merged**